storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -567,6 +567,39 @@
     public static final Object UI_HEADER_BUFFER_BYTES_SCHEMA = Number.class;
 
     /**
+     * This port is used by Storm DRPC for receiving HTTPS (SSL) DPRC requests from clients.
+     */
+    public static final String UI_HTTPS_PORT = "ui.https.port";
```

Comment:
Will be better if you could  modify file  
```
storm-core/conf/defaults.yaml  
```
and give these newly added configs default values.